### PR TITLE
chore: update header and footer package

### PR DESCRIPTION
**Pull request by leangseu-edx**
In light for adding renovate to the repo, I want to make sure that this doesn't block the future package update. Because header/footer have conflict peer dependency, it prevent `npm update` from running properlly.

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
approved

File: src/test/app.test.jsx [hunk 1]
```diff
@@ -213,7 +218,7 @@ describe('ESG app integration tests', () => {
     inspector = new Inspector(el);
   });
 
-  test('initialization', async (done) => {
+  test('initialization', async () => {
```
Comment (muselesscreator) [inline src/test/app.test.jsx:221]: Why are these getting removed from the integration test for this change?

Comment (leangseu-edx) [reply src/test/app.test.jsx:221]: I keep getting error telling me not to use async and done in the same test. It is redundant.

Comment (muselesscreator) [reply src/test/app.test.jsx:221]: I guess is new.  As long as the tests are passing, I don't really mind.  was intended to make the integration tests wait until the async test was actually done, which it wasn't before

**Pull request merged**